.github/workflows/python-linter.yml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+name: python-linter
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+      - 'v[0-9]+.[0-9]+.?[0-9]?*'
+      - ci/python-mypy-linter
+  pull_request:
+    branches:
+      - master
+      - 'v[0-9]+.[0-9]+.?[0-9]?*'
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  python_lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          base: master
+          list-files: shell
+          filters: |
+            python:
+              - 'lte/gateway/python/**'
+              - 'orc8r/gateway/python/**'
+      - uses: ricardochaves/python-lint@v1.4.0
+        if: steps.changes.outputs.python == 'true'
+        with:
+          python-root-list: ${{ steps.changes.outputs.python_files }}
```

Comment:
We could add more folders here I assume (e.g. add some modules that are already green)?

Reply:
This is one of the items I listed in the out-of-scope section above. Making modules green would be a follow up task for separate PRs, imho. This is for the basic CI at first.